README.md

```diff
@@ -0,0 +1,43 @@
+pip install -r requirements.txt
+$ ssh-keygen -o -a 100 -t ed25519 -f ~/.ssh/id_ed25519_gcp_compute_gcp-ansible -C "first.last@domain.com"
```

Comment:
Why not using this ansible module for creating the keys? https://docs.ansible.com/ansible/latest/modules/openssh_keypair_module.html

Reply:
we are
https://github.com/w3f/substrate-benchmarks/blob/init/w3f/substrate-benchmarks/roles/gcp/tasks/create.yml#L68-L73